File: Assets/Scripts/JavaScript/PlayerCasting.js

static var DistanceFromTarget: float;

var ToTarget: float;

function Update() {
    var hit: RaycastHit;
    if (Physics.Raycast(transform.position, transform.TransformDirection(Vector3.forward), hit)) {
        ToTarget = hit.distance;
        DistanceFromTarget = ToTarget;
    }
}

File: Assets/Scripts/Unused/PlayerCasting.cs
/*
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class PlayerCasting : MonoBehaviour {

    public float DistanceFromTarget;
    float ToTarget;
    	
	// Update is called once per frame
	void Update () {
        RaycastHit hit;

        if (Physics.Raycast(transform.position, transform.TransformDirection(Vector3.forward), out hit)) {
            ToTarget = hit.distance;
            DistanceFromTarget = ToTarget;
        }
	}

    public float getDistanceFromTarget() { return DistanceFromTarget; }                                     // Access the distance to a target in OpenDoubleDoors script
}
*/
